starteam/starcellbio_html
# Added yapf to enforce python code linting.

#### What are the relevant tickets?

Refs #594 
#### What's this PR do?

Introduces yapf, which will format python appropriately.
#### Where should the reviewer start?

Everything _except_ the python files.
#### How should this be manually tested?

You can check out master, run yapf on it (see Makefile for an example) then diff that and this branch.
#### Any background context you want to provide?
#### Screenshots (if appropriate)
#### What GIF best describes this PR or how it makes you feel?

![](http://s.mlkshk.com/r/17NQR.gif)
